events.py

```diff
@@ -1 +1,4 @@
 CLICK = 'click'
+HOVER = 'hover'
+TOOLTIP = 'tooltip'
```

Comment:
`TOOLTIP` is an odd name for an event. All the other events sound like events that are associated with an action.